packages/@react-spectrum/progress/stories/ProgressCircle.stories.tsx

```diff
@@ -32,50 +24,46 @@ const grayedBoxStyle: CSSProperties = {
 };
 
 storiesOf('Progress/ProgressCircle', module)
-  .addParameters({providerSwitcher: {status: 'positive'}})
-  .addDecorator(withKnobs)
+  .addParameters({
+    providerSwitcher: {status: 'positive'},
+    args: {value: 32},
+    argTypes: {
+      value: {
+        control: {
+          type: 'range',
+          min: 0,
+          max: 100
+        }
+      }
+    }
+  })
   .add(
     'Default',
-    () => render()
+    args => render(args)
```

Comment:
This story used to have a value of 0, but because of the default args for the other stories it uses value=32. This default change is okay, but feels weird.